language/phpmailer.lang-mg.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+* PHPMailer language file: refer to English translation for definitive list
+* French Version
```

Comment:
No, it's not French, it's Malgache, language spoken in Madagascar and other islands in the same region..